_core/smart/neon-node.md

```diff
@@ -27,64 +27,57 @@ git clone https://github.com/blockstack/stacks-blockchain.git
 cd stacks-blockchain
 ```
 
-Then build the project:
+### Configure your node to connect to the Neon network
 
-```bash
-cargo build
-```
+You'll need to update your node's configuration to connect to the public Neon testnet.
 
-Building the project on ARM:
-
-```bash
-cargo build --features "aarch64" --no-default-features
-```
+In the `stacks-blockchain` repository, open up the file `testnet/follower-config.toml`.
 
-### Setup your Proof-of-Burn miner
+First, find the `bootstrap_node` line in the `[node]` section, and update it to the following:
 
-In this phase of the testnet, miners will be burning BTC to participate in leader election. In order to mine, you'll need to configure your node with a BTC keychain, and you'll need to fund it with BTC. We're providing a BTC faucet to make it easy to run a miner.
+```toml
+bootstrap_node = "048dd4f26101715853533dee005f0915375854fd5be73405f679c1917a5d4d16aaaf3c4c0d7a9c132a36b8c5fe1287f07dad8c910174d789eb24bdfb5ae26f5f27@35.245.47.179:20444"
+```
 
-Let's start by generating a keypair:
+Next, find the `peer_host` line in the `[burnchain]` section, and update it to:
 
-```bash
-cargo run --bin blockstack-cli generate-sk --testnet
-
-# Output
-# {
-#  secretKey: "b8d99fd45da58038d630d9855d3ca2466e8e0f89d3894c4724f0efc9ff4b51f001",
-#  publicKey: "02781d2d3a545afdb7f6013a8241b9e400475397516a0d0f76863c6742210539b5",
-#  stacksAddress: "ST2ZRX0K27GW0SP3GJCEMHD95TQGJMKB7G9Y0X1MH"
-# }
+```toml
+peer_host = "35.245.47.179"
 ```
 
-**TODO**: guidance / update on generating a BTC address from this keypair.
+### Run your node
 
-Once you have your BTC address, head over to our testnet website to use the BTC faucet:
+You're all set to run a node that connects to the Neon network.
 
-**TODO**: URL, guidance for BTC faucet
+Back in the command line, run:
 
-Now that you have some testnet BTC, you'll need to configure your Stacks node to use this wallet.
+```bash
+cargo testnet ./testnet/follower-conf.toml
+```
 
-Open up the file `testnet/Stacks.toml`. Find the section that starts with `[burnchain]`. Update that section so that it looks like this:
+You should see some logs that look something like the this:
```

Comment:
I had a solid minute of installations coming before the logs listed below. Might want to call that out as well

Reply:
Added